Xamarin.Forms.Core/TabbedPage.cs

```diff
@@ -1,5 +1,10 @@
 using System;
+using Xamarin.Forms;
 using Xamarin.Forms.Platform;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-tabbedpage-bar-background-color", typeof(TabbedPage), nameof(TabbedPage.BarBackgroundColorProperty))]
+[assembly: StyleProperty("-xf-tabbedpage-bar-text-color", typeof(TabbedPage), nameof(TabbedPage.BarTextColorProperty))]
```

Comment:
oh, look, a similarity with NavigationPage. I think I see a pattern...

I wonder what could go wrong if we were unifying both ? in which scenario would they collide ? please discuss

Reply:
`-xf-bar-background-color`
`-xf-bar-text-color`